src/components/CioQuiz/quizLocalReducer.ts

```diff
@@ -92,6 +92,31 @@ export default function quizLocalReducer(
         isQuizCompleted: false,
       };
     }
+    case QuestionTypes.Skip: {
+      const { answers, answerInputs } = state;
+      const newAnswers = [...answers];
+      const lastAnswerInputIndex = answers.length;
+      const currentAnswerInput = Object.values(state.answerInputs)?.[lastAnswerInputIndex];
+      switch (currentAnswerInput.type) {
+        case QuestionTypes.OpenText:
+          newAnswers.push(['false']);
+          break;
+        case QuestionTypes.Cover:
+          newAnswers.push(['seen']);
+          break;
+        case QuestionTypes.SingleSelect:
+        case QuestionTypes.MultipleSelect:
+        default:
+          newAnswers.push([]);
+      }
+      return {
+        ...state,
+        // We now commit current answers to prevAnswerInputs
+        prevAnswerInputs: answerInputs,
```

Comment:
I'm not super familiar with this file's logic but should we be writing to `prevAnswerInputs` here?

Reply:
@mocca102 thoughts?

Reply:
What do you mean by? @esezen 

>  writing to prevAnswerInputs here?